README.md

```diff
@@ -16,3 +16,12 @@ We've started the process of bringing together various communities using this fo
 * Michael Ficarra ([@michaelficarra](https://github.com/michaelficarra))
 * Sebastian McKenzie ([Babel](https://github.com/babel/babel))
 * Kyle Simpson ([@getify](https://github.com/getify))
+
+# Philosophy
+
+Suggested additions and modifications must follow these guidelines:
+
+1. **Backwards compatible:** Non-additive modifications to existing constructs will not be considered unless immense support is in favor of such changes. ([eg. #65](https://github.com/estree/estree/issues/65))
+2. **Contextless:** Nodes should not retain any information about their parent. ie. a `FunctionExpression` should not be aware of if it's a concise method. (eg. [#5](https://github.com/estree/estree/issues/5))
+3. **Unique:** Information should not be duplicated. ie. a `kind` property should not be present on `Literal` if the type can be discerned from the `value`. (eg. [#61](https://github.com/estree/estree/issues/61))
+4. **Extensible:** New nodes should be specced to easily allow future spec additions. This means expanding the coverage of node types. ie. `MetaProperty` over `NewTarget` to cover future meta properties. (eg. [#32](https://github.com/estree/estree/pull/32))
```

Comment:
Suggested additions:
* **Liberal**: Nodes impose no restrictions on their children beyond type and no restrictions at all on deeper descendants. Many conforming ESTree objects represent invalid ECMAScript code, and consumers for which this is problematic must handle context-sensitive restrictions (strict mode, required/prohibited ancestor nodes, name uniqueness/validity, etc.) theirselves.
* **Portable**: Structures should be losslessly serializable into a variety of formats (but most notably, JSON), and therefore should be acyclic and utilize only object, array, string, number, boolean, and null data types.
* **Concise**: Updating existing types is preferred over introducing new types, and introducing new subtypes is preferred over introducing slightly-modified copies of existing types.